### DISCUSSION TITLE
extend network with wifi

### DISCUSSION THREAD

**Discussion by enthalpie91** [Koenkk/zigbee2mqtt#23984] [General]
HI 
I am looking for zigbee and as i understand the network is extanded with some "devices" acted as router zigbee.  But if somes devices ares too far from the zigbee network i can't add this on. 
Is a possibility to extend the zigbee network with a device (second hub)  talking with the main network hub from wifi and so possible to connect a far from room
Best

**Comment by chris-1243** [comment]
The Zigbee network is a mesh. You start with the coordinator and to extend it, you add routers (like lights, plugs).

https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html#adding-routers-to-your-network

In order to extend your network, add some routers. Coordinators (USB, LAN) may be flashed as router and are great to extend your network. 

It depends how far is your room. I would not recommend to connect a coordinaror via [wifi](https://www.zigbee2mqtt.io/advanced/remote-adapter/connect_to_a_remote_adapter.html)

**Comment by enthalpie91** [comment]
Hello
First at all thanks for your answer
second i did'nt know all about device and for me at this state,  you have only repeteur , that's means the apparatus should see the zigbee network to be able to extend the signal . but if the other room where i want to install more device is out of the range of the zigbee signal i am looking for 2 routers zigbee/wifi or zigbee/lan .

**Comment by enthalpie91** [comment]
[zigbee lan.odg](https://github.com/user-attachments/files/17019497/zigbee.lan.odg)

**Comment by chris-1243** [comment]
Zigbee Lan/POE coordinator:

https://www.zigbee2mqtt.io/guide/adapters/zstack.html

https://www.zigbee2mqtt.io/guide/adapters/emberznet.html

**Reply by enthalpie91** [reply]
OK so it is possible with these
i will look at
thank's
